### DISCUSSION TITLE
Tempo-Search: Pre-fill "Find" textbox in trace view with searched tag

### DISCUSSION THREAD

**Discussion by BitProcessor** [grafana/grafana#41640] [Observability]
I searched for a tag, next opened a trace from the search result.

Would be very nice if the `Find` textbox is pre-filled with the tag I just searched for.
That would highlight the lines in the trace overview where the tag appears.

![Screenshot 2021-11-12 at 13 23 30](https://user-images.githubusercontent.com/50638541/141466985-ff81ed30-d6e9-4233-861f-443a22307be4.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
